src/it/resources/org/checkstyle/suppressionxpathfilter/annotationusestyle/SuppressionXpathRegressionAnnotationUseStyleFive.java

```diff
@@ -0,0 +1,11 @@
+package org.checkstyle.suppressionxpathfilter.annotationusestyle;
+
+@Ann(value={"foo", "bar"}) //warn
+public class SuppressionXpathRegressionAnnotationUseStyleFive {
+
+}
+
+@interface Ann {
```

Comment:
No need to use custom annotation, `@SuppressWarnings` is fine

Reply:
Done